dnsforward/dnsforward.go

```diff
@@ -26,6 +26,8 @@ const (
 	parentalBlockHost     = "family-block.dns.adguard.com"
 )
 
+const unspecifiedAddress = "0.0.0.0"
```

Comment:
Please use `net.IPv4zero` instead of this.
This will also allow us to remove `net.ParseIP()` call below and pass it directly to `genARecord()`.

Reply:
@szolin didn't know about `net.IPv4zero`. Fixed.

Also added a test for the null filter.

Not sure if it's best practice to hardcode the value like `null_ip`, or if it should be added as a constant.